maven-python-plugin/README.md

```diff
@@ -78,7 +78,7 @@ If a version number is specified in the setup.py it will override both options a
 `test` phase:
```

Comment:
What do you think about adding a line with `mvn clean compile` explaining that will run static typing analysis using mypy (and that it's the only supported python static typing lib for now) in the "Build your python code with maven" section?  I'm not sure having only an additional configuration parameter section is explicit enough

Reply:
Good point. I forgot to document it. Fixed in 1f01e3751ad1